tests/setup_fake_db.py

```diff
@@ -21,7 +21,7 @@
 
 from new_arrivals_chi.app.main import create_app, db
 from new_arrivals_chi.app.utils import setup_logger
-from db_test import create_fake_data
+from .db_test import create_fake_data
```

Comment:
What command are you using to run your code? I'm not sure we need a relative import? Besides, the change breaks the code for me:

Using relative import
```
poetry run python3 -m setup_fake_db.py
Traceback (most recent call last):
  File "<frozen runpy>", line 189, in _run_module_as_main
  File "<frozen runpy>", line 112, in _get_module_details
  File "/Users/aaronhaefner/chicago_projects/new-arrivals-chi/tests/setup_fake_db.py", line 24, in <module>
    from .db_test import create_fake_data
ImportError: attempted relative import with no known parent package
```

No relative import
```
poetry run python3 -m setup_fake_db
2024-05-10 01:30:50,701 - FakeDataCreator - INFO - Starting the application setup for fake data creation.
2024-05-10 01:30:50,719 - FakeDataCreator - INFO - Starting to create fake data
2024-05-10 01:30:50,739 - FakeDataCreator - INFO - Added org, user, location & hours for user 30
2024-05-10 01:30:50,739 - FakeDataCreator - INFO - Fake data creation completed successfully
2024-05-10 01:30:50,739 - FakeDataCreator - INFO - Fake data has been created successfully.
```

Reply:
Hi Aaron, I get an error without a relative import and that's why I added it. I am in poetry shell and also ran poetry install. I'll continue to debug. This is what I see:

`================================================ERRORS =================================================
_____________________________ ERROR collecting tests/sql_injection_test.py ______________________________
ImportError while importing test module '/Users/xiomarasalazar/Documents/UChicago/Spring_2024/SWE_Civic_tech/Github/new-arrivals-chi/tests/sql_injection_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
sql_injection_test.py:45: in <module>
    from db_test import create_fake_user, create_fake_organization
E   ModuleNotFoundError: No module named 'db_test'
======================================== short test summary info ========================================
ERROR sql_injection_test.py`

Reply:
Write out the full path
```
from tests.db_test import create_fake_user, create_fake_organization
from tests.setup_fake_db import main
```

Reply:
In `setup_fake_db.py` it should be
```
from tests.db_test import create_fake_data
```

Reply:
This solved it! Ty!